src/mixer/basetrackplayer.h

```diff
@@ -100,6 +107,10 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
     ControlObjectSlave* m_pPreGain;
     ControlObjectSlave* m_pSpeed;
     ControlObjectSlave* m_pPitchAdjust;
+    QScopedPointer<ControlObjectSlave> m_pInputEnabled;
+    QScopedPointer<ControlObjectSlave> m_pPassthroughEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlStatus;
```

Comment:
> This QScopedPointer member variables are consuming much more cpu cycles, since we
> call the default constructor fist and then we call reset to adopt the final pointer.

I don't consider default constructing a pointer -- a single XOR instruction -- "much more" :) 

If I were to write a raw pointer as a member variable and use it as I did here, I would initialize it to null first to prevent accidental use of uninitialized memory -- so there is no difference between how I would have written it with a raw pointer or scoped pointer.